src/Avalonia.Visuals/Platform/IGeometryImpl.cs

```diff
@@ -47,5 +53,38 @@ public interface IGeometryImpl
         /// <param name="transform">The transform.</param>
         /// <returns>The cloned geometry.</returns>
         ITransformedGeometryImpl WithTransform(Matrix transform);
+
+        /// <summary>
+        /// Attempts to get the corresponding point from the
```

Comment:
```suggestion
        /// Attempts to get the corresponding point at the
```
I wonder if that won't make more sense grammatically :)